.travis.yml

```diff
@@ -1,21 +1,18 @@
 ---
-sudo: false
-language: ruby
-cache: bundler
+sudo: required
+services: docker
+addons:
+  apt:
+    packages:
+    - docker-ce
 matrix:
   include:
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=master
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=7.0
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=6.7
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=6.6
-  - rvm: jruby-1.7.27
-    env: LOGSTASH_BRANCH=5.6
+  - env: SNAPSHOT=true ELASTIC_STACK_VERSION=8.x
+  - env: SNAPSHOT=true ELASTIC_STACK_VERSION=7.x
+  - env: ELASTIC_STACK_VERSION=7.x
+  - env: SNAPSHOT=true ELASTIC_STACK_VERSION=6.x
```

Comment:
nitpick: for some reason these would make a little more sense to me if we always specified `ELASTIC_STACK_VERSION` before the optional `SNAPSHOT` one. I think it visually lets me acknowledge the versions we're testing and then layer on "_oh yeah, we also want the snapshot version since we're in active development_":

~~~
matrix:
  include:
  - env: ELASTIC_STACK_VERSION=8.x SNAPSHOT=true
  - env: ELASTIC_STACK_VERSION=7.x SNAPSHOT=true
  - env: ELASTIC_STACK_VERSION=7.x
  - env: ELASTIC_STACK_VERSION=6.x SNAPSHOT=true
  - env: ELASTIC_STACK_VERSION=6.x
  - env: ELASTIC_STACK_VERSION=5.x
~~~